nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -191,7 +191,7 @@
         <dependency>
             <groupId>com.exceptionfactory.socketbroker</groupId>
             <artifactId>socketbroker</artifactId>
-            <version>1.1.0</version>
+            <version>1.3.0</version>
```

Comment:
This change is for a different library and should be reverted.

Reply:
@exceptionfactory thanks. Change reverted.